includes/module/settings/StandardSettings.php

```diff
@@ -144,7 +144,8 @@ public function generateForm()
                 'name' => 'MERCADOPAGO_STANDARD_MODAL',
                 'is_bool' => true,
                 'desc' => $this->module->l(
-                    'Your customers will access the Mercado Pago payment form without leaving your store. If you deactivate it, they will be redirected to another page.',
+                    'Your customers will access the Mercado Pago payment form without leaving your store.' .
```

Comment:
A traduçao nao será encontrada. Deve concatenar por function:
               $this->module->l(' Activating it can affect fraud prevention. ', 'StandardSettings') .
               $this->module->l('Leave it inactive so we can ', 'StandardSettings') .
               $this->module->l('take care of your charges', 'StandardSettings'),